ldaptor/test/test_pureber.py

```diff
@@ -30,9 +30,11 @@ def s(*l):
         r = r + str(e)
     return r
 
+
 def l(s):
     """Split a string to ord's of chars."""
-    return map(lambda x: ord(x), s)
+    return [six.byte2int([x]) for x in s]
```

Comment:
> (line 36) - Is there some reason for return [six.byte2int([x]) for x in s] instead of return [six.byte2int(x) for x in s]? I'm not sure why the extra list ...

I think that byte2int requires a list https://pythonhosted.org/six/#six.byte2int as it needs a byte string as input

Reply:
I'm pretty certain `bs` is just a byte string, though a list with a single byte char seems to work:

	waldbiec@waldbiec-Latitude-E5570:~/git-repos/ldaptor$ /usr/bin/python2 -V
	Python 2.7.12
	waldbiec@waldbiec-Latitude-E5570:~/git-repos/ldaptor$ /usr/bin/python3 -V
	Python 3.5.2
	waldbiec@waldbiec-Latitude-E5570:~/git-repos/ldaptor$ cat test.py
	#! /usr/bin/env python

	import six


	def main():
	    chars = b'chars'
	    l = [chars[0]]
	    print(six.byte2int(chars))
	    print(six.byte2int(l))

	if __name__ == "__main__":
	    main()
	waldbiec@waldbiec-Latitude-E5570:~/git-repos/ldaptor$ /usr/bin/python2 test.py 
	99
	99
	waldbiec@waldbiec-Latitude-E5570:~/git-repos/ldaptor$ /usr/bin/python3 test.py 
	99
	99

Unless there's something subtle here that I'm missing.